runtime/doc/news.txt

```diff
@@ -18,6 +18,12 @@ BREAKING CHANGES IN HEAD                                    *news-breaking-dev*
 The following changes to UNRELEASED features were made during the development
 cycle (Nvim HEAD, the "master" branch).
 
+MAIN
+
+* On Windows, filename arguments on the command-line prefixed with "~/" or
+  "~\" are now expanded to the user's profile directory, not a relative path
+  to a literal "~" directory.
+
```

Comment:
This is the wrong place to put this. It should be in the "EDITOR" subsection of the "NEW FEATURES" section.